<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpendsByYears</class>
 <widget class="QMainWindow" name="SpendsByYears">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>911</width>
      <height>611</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>931</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Spends by years</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="graph">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>75</y>
       <width>881</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>550</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total cost:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalCost">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>515</width>
      <height>56</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="management">
       <property name="text">
        <string>Management</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statistics">
       <property name="text">
        <string>Stastistics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signOut">
       <property name="text">
        <string>Sign out</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>221</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="salesByYears">
       <property name="text">
        <string>Sales by years</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salesByEmployees">
       <property name="text">
        <string>Sales by employees</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salesByProducts">
       <property name="text">
        <string>Sales by products</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spendsByYears">
       <property name="text">
        <string>Spends by years</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spendsBySuppliers">
       <property name="text">
        <string>Spends by suppliers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spendsByProducts">
       <property name="text">
        <string>Spends by products</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
